notebooks/official/custom/get_started_with_vertex_endpoint_and_shared_vm.ipynb

```diff
@@ -415,9 +380,9 @@
       "source": [
         "## Set up clients\n",
         "\n",
-        "The Vertex  works as a client/server model. On your side (the Python script) you will create a client that sends requests and receives responses from the Vertex AI server.\n",
+        "The Vertex  works as a client/server model. On your side (the Python script) you create a client that sends requests and receives responses from the Vertex AI server.\n",
         "\n",
-        "You will use different clients in this tutorial for different steps in the workflow. So set them all up upfront.\n",
+        "You use different clients in this tutorial for different steps in the workflow. So set them all up upfront.\n",
```

Comment:
change:
- "So set them all up upfront" 
to:
- "This means you have to set them all up at the start."